public/src/components/app.js

```diff
@@ -16,3 +20,18 @@ export default class App extends Component {
     );
   }
 }
+
+function mapStateToProps(state) {
+  return {
+    auth: state.auth,
+    account_info: state.account_info
+  };
+}
+
+function mapDispatchToProps() {
+  return { };
+}
+
+export default withRouter(
+  connect(mapStateToProps, mapDispatchToProps)(App)
```

Comment:
I notice mapDispatchToProps() returns an empty object, can you leave this blank in connect?